internal/exec/aws_eks_update_kubeconfig.go

```diff
@@ -222,14 +223,14 @@ func ExecuteAwsEksUpdateKubeconfig(kubeconfigContext cfg.AwsEksUpdateKubeconfigC
 		args = append(args, fmt.Sprintf("--region=%s", region))
 	}
 
-	err = ExecuteShellCommand("aws", args, shellCommandWorkingDir, nil, dryRun, true, "")
+	err = ExecuteShellCommand(cliConfig, "aws", args, shellCommandWorkingDir, nil, dryRun, true, "")
 	if err != nil {
 		return err
 	}
 
 	if kubeconfigPath != "" {
 		message := fmt.Sprintf("\n'kubeconfig' has been downloaded to '%s'\nYou can set 'KUBECONFIG' ENV var to use in other scripts\n", kubeconfigPath)
-		u.PrintInfo(message)
+		u.LogInfo(cliConfig, message)
```

Comment:
This makes sense as Info. It's pertinent the execution of the command and the user's expectations. The user needs this "info" to proceed.